test/integration/puppetlabs/puppetserver/testutils.clj

```diff
@@ -331,3 +331,30 @@
    (with-puppet-conf puppet-conf-file conf-dir))
   ([puppet-conf-file dest-dir]
    (with-puppet-conf-fixture {"puppet.conf" puppet-conf-file} dest-dir)))
+
+(defn copy-config-files
+  [dirs-to-copy]
+  (doseq [[src target] dirs-to-copy]
+    (fs/mkdirs target)
+    (let [[_ dirs files] (first (fs/iterate-dir src))]
+      (doseq [d dirs] (fs/copy-dir (fs/file src d) target))
+      (doseq [f files] (fs/copy (fs/file src f) (fs/file target f))))))
+
+(defn remove-config-files
+  [dirs-to-copy]
+  (doseq [[src target] dirs-to-copy]
+    (let [[_ dirs files] (first (fs/iterate-dir src))]
+      (doseq [d dirs] (fs/delete-dir (fs/file target d)))
+      (doseq [f files] (fs/delete (fs/file target f))))))
+
+(defmacro with-puppet-config-files
```

Comment:
That sounds good to me - I think the new name is better. I still question exactly how useful it is to have both of these functions - mostly I think in the past we've had a tendency to create test helper functions all over the place, and it gets a little out of control. I just want to make sure we don't recreate functionality we already have. Anyway, I'm good with this as-is.